content/en/blog/2022-05-05-introducing-kluctl-and-flux.md

```diff
@@ -0,0 +1,83 @@
+
+---
+title: "Kluctl And Flux"
+linkTitle: "Kluctl And Flux"
+date: 2022-05-05
+author: Alexander Block (@codablock)
+---
+
+We're very happy to announce that Kluctl can from now on be used together with [Flux](https://fluxcd.io/). This
+will allow you to combine the workflows and features advertised by Kluctl with GitOps style continuous delivery.
+
+## GitOps vs Kluctl
+One of the first questions that we usually get when introducing Kluctl to someone is something like:
+"Why not GitOps?" or "Why not Flux?". There seems to be a common misunderstanding that arises in many people
+when trying to understand Kluctl on first sight, which is to believe that Kluctl is an alternative or competitor 
+to GitOps and Flux (or even ArgoCD).
+
+This is not the case. If one wants to compare Kluctl with something else, then it's more appropriate to compare it
+to Helm, Kustomize or Helmfile. It should be clear that Kustomize for example is not an alternative/competitor for Flux,
+but instead an essential tool and building block to make it work.
+
+Kluctl can be looked at from the same perspective when it comes to Flux. Flux implements Helm and Kustomize support
+via different controllers, namely the [kustomize-controller](https://fluxcd.io/docs/components/kustomize/) and the
+[helm-controller](https://fluxcd.io/docs/components/helm/). Kluctl can simply do the same to support Flux.
+
+## Introducing the Kluctl Flux Controller
+An alpha version of the [Kluctl Flux Controller]({{< ref "docs/flux" >}}) has just been released. It allows to
+create [KluctlDeployment]({{< ref "docs/flux/kluctldeployment" >}}) objects which are reconciled in a similar
+fashion as [Kustomizations](https://fluxcd.io/docs/components/kustomize/kustomization/).
+
+Each KluctlDeployment specifies a source object (e.g. a [GitRepository](https://fluxcd.io/docs/components/source/gitrepositories/)),
+the [target]({{< ref "docs/reference/kluctl-project/targets" >}}) to be deployed and some information on how
+to handle kubeconfigs. The controller then regularly reconciles the deployment.
```

Comment:
The controller then regularly reconciles the deployment and on changes fire a `kluctl deploy` to the enviornment